wdl/imp_sub.wdl

```diff
@@ -4,17 +4,19 @@ workflow imputation {
     Array[String] beds
     String joint_qc_exclude_variants
     Array[String] batch_qc_exclude_variants
+    String force_impute_variants
     String exclude_samples
+    Map[Int, String] ref_panel
     Int n_batches = length(batch_qc_exclude_variants)
 
     File? exclude_denials
 
     scatter (i in range(n_batches)) {
         call plink_to_vcf {
-            input: bed=beds[i], joint_qc_exclude_variants=joint_qc_exclude_variants, batch_qc_exclude_variants=batch_qc_exclude_variants[i], exclude_samples=exclude_samples
+            input: bed=beds[i], joint_qc_exclude_variants=joint_qc_exclude_variants, batch_qc_exclude_variants=batch_qc_exclude_variants[i], exclude_samples=exclude_samples, force_impute_variants=force_impute_variants
```

Comment:
lets try to start having shorter lines for easier reading i.e. add line change at a suitable spots